data/table.txt

```diff
@@ -7785,9 +7827,11 @@
 冟 冟 1 0 0 0 1 0 0 0 0 bhap bhap NA 0
 冠 冠 1 1 0 0 1 0 0 0 0 bmui bmui NA 21405
 冡 冡 1 0 0 0 1 0 0 0 0 bmmo bmmo NA 0
-冢 冢 1 1 0 0 1 0 0 0 0 bmmo,gbmo bmso,xbmso NA 20766
+冢 冢 1 1 0 0 1 0 0 0 0 bmmo bmso,xbmso NA 20766
```

Comment:
@yookoala [said](https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099623):

> "xbmso" returns nothing on MacOS. Should be non-standard mapping.

---

So for `bmso` we have the following characters:

```
豚 豚 1 1 0 0 1 0 0 0 0 bmso bmso NA 19356
䐁 䐁 1 0 1 0 0 0 0 0 0 bmso bmso NA 0
冢 冢 1 1 0 0 1 0 0 0 0 bmmo bmso,xbmso NA 20766
```

Isn't this a case of x-disambiguation?

Reply:
So Wan Leung just said that Mac OS X uses the short-lived Cangjie version 4, not version 5. :open_mouth:

Discussing this with @yookoala, we decided to ignore all those comments about Mac.